packages/file_selector/file_selector_android/pubspec.yaml

```diff
@@ -0,0 +1,34 @@
+name: file_selector_android
+description: Android implementation of the file_selector plugin.
+repository: https://github.com/flutter/plugins/tree/main/packages/file_selector/file_selector_android
+issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+file_selector%22
+version: 0.5.0
+
+publish_to: 'none' # Remove this line when we make the first publish to pub.dev
```

Comment:
This should be removed; landing this PR would be when we would want to publish to pub.dev.